src/firecrest/status/commands/id_command.py

```diff
@@ -12,8 +12,9 @@ class IdCommand(BaseCommandWithTimeout):
 
     def __init__(
         self,
+        command_timeout: int = 5
```

Comment:
You also need to update the status router to set the actual timeout:
https://github.com/eth-cscs/firecrest-v2/blob/415f79204fe9a2c3d01434b416639af3a8cd7ee2/src/firecrest/status/router.py#L173

Reply:
Perfect, I will update the PR. Thanks for catching this.